e3sm_diags/driver/lat_lon_driver.py

```diff
@@ -87,45 +91,20 @@ def run_diag(parameter):
             parameter.var_id = var
 
             mv1 = test_data.get_climo_variable(var, season)
-            mv2 = ref_data.get_climo_variable(var, season)
+            try:
+                mv2 = ref_data.get_climo_variable(var, season)
+            except Exception:
+                mv2 = mv1
```

Comment:
```suggestion
            try:
                mv2 = ref_data.get_climo_variable(var, season)
            except RuntimeError:
                mv2 = mv1
```

As Xylar noted, it is best practice to be as specific as possible with Python exceptions (https://docs.python.org/3/library/exceptions.html).

The `.get_climo_variable()` function raises  `RuntimeError`  in multiple conditionals:

https://github.com/E3SM-Project/e3sm_diags/blob/b15a05feb08539e9c71ef0628720a90a415c4dc0/e3sm_diags/driver/utils/dataset.py#L128-L131

https://github.com/E3SM-Project/e3sm_diags/blob/b15a05feb08539e9c71ef0628720a90a415c4dc0/e3sm_diags/driver/utils/dataset.py#L162-L166

_Side-note: The exceptions in `.get_climo_variable()` should be `TypeError` instead of `RuntimeError`._